features/yield/FarmingCalculator.tsx

```diff
@@ -225,6 +213,9 @@ const FarmingCalculator = () => {
           // Reward has not started.
           continue;
         } else {
+          if (reward.token == "UMA") {
+            reward.count = liquidityRewards;
```

Comment:
just updating the hardcoded reward count with our dynamically calculated liquidity rewards from devmining. This is what gets displayed in the UI